pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = [
+    "setuptools >= 51",
```

Comment:
does this have to be so restrictive? I think setuptools >=45 should be more than enough

Reply:
Ah, it's a typo in the `setuptools` docs for when `entry_points` was first supported in `setup.cfg` -- digging deeper, it is actually much earlier, so 45 is fine